.gitignore

```diff
@@ -0,0 +1,2 @@
+/pkg/

```

Comment:
I think it's best if you remove the `.gitignore` since it is only a problem for your setup. Try [this technique](https://help.github.com/articles/ignoring-files/#explicit-repository-excludes) to ignore files without having a `.gitignore`.